.github/workflows/run-codegen-test.yml

```diff
@@ -65,14 +82,13 @@ jobs:
         with:
           ref: ${{ env.PR_COMMIT_SHA }}
       - name: Test Downstream
-        uses: pulumi/action-test-provider-downstream@releases/v7
+        uses: pulumi/action-test-provider-downstream@friel/v8
```

Comment:
Can we get this merged to the downstream repo proper rather than running off a branch. Also we really ought to tighten that repo up to run of main and have tags.

Reply:
I can make the PR and then make a tag

Reply:
Now targeting `@v1.0.0` released tag.